### PR TITLE
Parse celery pings

### PR DESCRIPTION
**Pull request by benrudolph**
@gcapalbo here's a version using celery pings. going to throw it up on staging

cc: @orangejenny

### CODE REVIEW

File: corehq/apps/hqadmin/utils.py [hunk 1]
```diff
@@ -87,6 +87,17 @@ def get_task_html(detailed_stats, worker_name):
...
 
 
+def parse_celery_pings(worker_responses):
+    pings = {}
+    for worker in worker_responses:
+        assert len(worker.keys()) == 1 and '@' in worker.keys()[0]
+
+        worker_fullname = worker.keys()[0]
+        worker_hostname = worker_fullname.split('@')[1]
+        pings[worker_hostname] = worker[worker_fullname]['ok'] == 'pong'
```
Comment (gcapalbo) [inline corehq/apps/hqadmin/utils.py]: we might want to use `worker[worker_fullname].get('ok')` just to be safe so we never fail hard

**Comment by gcapalbo** [comment]
looks great, thanks. just one minor comment otherwise i think we're good to merge once it's tested on staging

File: corehq/apps/hqadmin/utils.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/hqadmin/utils.py]: 👍 https://github.com/dimagi/commcare-hq/pull/13535/commits/c545efdfee2ba691500351d0e881a26953e94941

**Comment by gcapalbo** [comment]
good to merge from my end, let me know once you're comfortable with the staging test

**Comment by benrudolph** [comment]
caught one dumb mistake, but should be good to go now

**Pull request merged**